src/libAtomVM/term.c

```diff
@@ -239,7 +239,7 @@ static int term_type_to_index(term t)
     }
 }
 
-int term_compare(term t, term other, Context *ctx)
+int term_compare(term t, term other, GlobalContext *glb)
```

Comment:
Same point here, I agree about this change but it adds a lot of noise on top of something already quite complex.

Reply:
This was eventually done in fa9752a54